package.json

```diff
@@ -48,15 +48,15 @@
     "@types/base-64": "^1.0.2",
     "aws-kms-signer": "^0.5.3",
     "base-64": "^1.0.0",
-    "body-parser": "^1.20.2",
+    "body-parser": "^1.20.3",
     "bullmq": "^5.11.0",
-    "cookie": "^0.5.0",
-    "cookie-parser": "^1.4.6",
+    "cookie": "^0.7.0",
+    "cookie-parser": "^1.4.7",
```

Comment:
i would recommend using strict versions here rather than `^` ones. you never know what patch update is gonna get sneaked in there otherwise

Reply:
Hmm I think this is a mixed opinion, will ping the channel for more thoughts.

Updating minor versions gets us security/performance improvements, but you're right it can introduce unintentional breaking changes.